src/js/components/Carousel/doc.js

```diff
@@ -50,6 +55,11 @@ export const themeDoc = {
     type: 'string',
     defaultValue: undefined,
   },
+  'carousel.disabled.icons.color': {
+    description: 'The color used for disabled Carousel icons.',
+    type: 'string',
```

Comment:
I think `color` may be a type of an object as well.  i.e `{dark: undefined, light: undefined}`, can you verify the object use cases works?